.env

```diff
@@ -1 +0,0 @@
-DATABASE_URL = mongodb+srv://derttor:oN53Ohrjj20B12bM@cluster-task-manager.xobp0bj.mongodb.net/task-manager?retryWrites=true&w=majority
```

Comment:
Credentials still can be found in git. We need to re-write git history to remove it completely. Ping me if help is needed on this point.